app/views/devise/shared/_error_messages.html.haml

```diff
@@ -0,0 +1,9 @@
+- if resource.errors.any?
```

Comment:
if you haven't modified this please remove it

Reply:
We didn't modify it, but this code is what renders the errors when the requirements for the signup is not fulfilled, so we need this for the sad-path

Reply:
But that's the thing with devise and gems like this overall. You will have access to this functionality without having this file in the code base. You only need those views if you need to change anything.  

Try it out without this file